components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/import-file-dialog.js

```diff
@@ -16,266 +16,193 @@
 
-                //Gets the selected location from tree and sets the value as location
-                this.listenTo(fileBrowser, 'selected', function (selectedLocation) {
-                    selectedFilePath = selectedLocation;
-                    var pathAttributes = selectedLocation.split(self.pathSeparator);
-                    var fileName = _.last(pathAttributes);
+                        if (existsResponse.error === undefined) {
+                            if (existsResponse.exists) {
+                                alertError("A file already exist in workspace with selected name.");
+                                return;
+                            } else {
+                                reader.readAsText(file);
+                            }
+                        } else {
+                            alertError("Error in reading the file.");
+                        }
 
-                    if(selectedLocation && fileName.lastIndexOf(".siddhi") != -1){
-                        location.val(fileName);
+                        reader.onload = (function (reader) {
+                            return function () {
+                                var fileContent = reader.result;
+                                importConfiguration(fileContent);
+                            }
+                        })(reader);
                     }
-                });
 
-
-                fileImport.find("button").filter("#importButton").click(function () {
-
-                    var _location = selectedFilePath;
-                    if (_.isEmpty(fileImport.find("input").filter("#location"))) {
-                        openFileWizardError.text("Invalid Value for Location.");
-                        openFileWizardError.show();
-                        return;
+                    function existFileInPath(options) {
+                        var client = self.app.workspaceManager.getServiceClient();
+                        var data = {};
+                        var workspaceServiceURL = app.config.services.workspace.endpoint;
+                        var saveServiceURL = workspaceServiceURL + "/exists/workspace";
+                        var payload = "configName=" + btoa("workspace" + app
+                            .getPathSeperator() + options.configName);
+
+                        $.ajax({
+                            type: "POST",
+                            contentType: "text/plain; charset=utf-8",
+                            url: saveServiceURL,
+                            data: payload,
+                            async: false,
+                            success: function (response) {
+                                data = response;
+                            },
+                            error: function (xhr, textStatus, errorThrown) {
+                                data = client.getErrorFromResponse(xhr, textStatus, errorThrown);
+                                log.error(data.message);
```

Comment:
This will send the error message to browser's console and will log in the log file.  And this will be caught at line 78 and send an "alertError" as well.